client/src/Dashboard/utils.js

```diff
@@ -0,0 +1,30 @@
+export function getPRFromRepos(repos) {
+	return repos.reduce((acc, element) => {
+		return acc.concat(element.pullRequests.nodes);
+	}, []);
+}
+
+export function getReposFromOrgs(orgs) {
+	return orgs.reduce(
+		(acc, element) => acc.concat(element.repositories.nodes),
+		[]
+	);
+}
+
+export function mergePRs(queryData) {
```

Comment:
Totally agree, we can use kind of `concatPRs`, `joinPRs` since the group could be confusing by "groupBy" which means split the PRs by any criteria. But please, feel free to rename it following the conventions of the project :)